core-services/image-mirroring/edge-infrastructure/mapping_edge_infrastructure_recert

```diff
@@ -0,0 +1 @@
+registry.ci.openshift.org/edge-infrastructure/latest:recert quay.io/edge-infrastructure/recert:latest
```

Comment:
```suggestion
registry.ci.openshift.org/edge-infrastructure/recert:latest quay.io/edge-infrastructure/recert:latest
```
as ``podman pull registry.ci.openshift.org/edge-infrastructure/recert:latest`` works and ``podman pull registry.ci.openshift.org/edge-infrastructure/latest:recert`` doesn't

Reply:
Big typo mistake, thanks for spotting this :+1: